src/orb.yml

```diff
@@ -391,6 +415,10 @@ jobs:
         description: Use yarn to install NPM modules instead of npm.
         type: boolean
         default: false
+      working_directory:
+        description: Directory containing package.json
```

Comment:
since this is top level parameter, could you describe its purpose a little bit more here?